app/assets/javascripts/angular/controllers/orders_controller.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+var app = angular.module('taxi', ['ngRoute']);
+
+app.controller('CreateOrderController', ['$scope', '$http', function($scope, $http) {
+    $scope.phone_pattern = /(0)[0-9]{9}/;
+    $scope.email_pattern = /^(.+)@(.+)$/;
+
+    $scope.addOrder = function() {
+    if ($scope.order.email == undefined) {
```

Comment:
Use strict checks for equality `===`
In this case it is sufficient to write `if (!$scope.order.email)`